File: src/WebClient/WareHouse/src/app/entity/WareHouseLimit.ts
import { BaseEntity } from "./BaseEntity";
import { Unit } from "./Unit";
import { WareHouse } from "./WareHouse";
import { WareHouseItem } from "./WareHouseItem";

export interface WareHouseLimit extends BaseEntity {
    wareHouseId: string;
    itemId: string;
    unitId: string;
    unitName: string;
    minQuantity: number;
    maxQuantity: number;
    createdDate: string;
    createdBy: string;
    modifiedDate: string;
    modifiedBy: string;
    item: WareHouseItem;
    unit: Unit;
    wareHouse: WareHouse;
}

File: src/Services/WareHouse/WareHouse.Domain/Entity/WareHouseLimit.cs
using System;
using System.Collections.Generic;
using Share.BaseCore;
#nullable disable

namespace WareHouse.Domain.Entity
{
    public partial class WareHouseLimit : BaseEntity
    {
        public string WareHouseId { get; set; }
        public string ItemId { get; set; }
        public string UnitId { get; set; }
        public string UnitName { get; set; }
        public int MinQuantity { get; set; }
        public int MaxQuantity { get; set; }
        public DateTime CreatedDate { get; set; }
        public string CreatedBy { get; set; }
        public DateTime ModifiedDate { get; set; }
        public string ModifiedBy { get; set; }

        public virtual WareHouseItem Item { get; set; }
        public virtual Unit Unit { get; set; }
        public virtual WareHouse WareHouse { get; set; }
    }
}
